packages/functional-tests/tests/signin/redirect.spec.ts

```diff
@@ -17,76 +15,44 @@ test.describe('severity-2 #smoke', () => {
         // skip these tests if the flag is on.
         test.skip(config.showReactApp.signUpRoutes === true);
 
-        signUpReactEnabled = config.showReactApp.signUpRoutes === true;
```

Comment:
I wasn't really sure what to do about this contradiction and opted to remove the if conditions associated. Perhaps that's not correct?

Also the tests seem to be backbone/react agnostic? I am able to run these tests against the react pages (skip condition removed).

Reply:
TLDR;: Makes sense to me to skip this set of tests entirely for React - we should file a ticket to create a new set of tests for React since there are flow differences. The fact that this test passes when the flag is enabled is misleading because the test will actually run through backbone unless we add the `forceExperiment` params.

If this test is enabled for React with Signup routes set to fullProdRollout, these tests fail.